naga/src/front/wgsl/parse/directive/language_extension.rs

```diff
@@ -61,12 +63,24 @@ impl LanguageExtension {
 }
 
 /// A variant of [`LanguageExtension::Implemented`].
-#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
-pub(crate) enum ImplementedLanguageExtension {}
```

Comment:
```
 65 | /// A variant of [`LanguageExtension::Implemented`].
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this item is private
```
the problem is that we still only expose `ImplementedLanguageExtension` outside of this module, hence rustdoc errors. We will either need to expose them all or change it's doc (I did the second option initially).

Reply:
I think exposing all `*LanguageExtension` APIs is a fine course of action. However, I think I'd prefer we not rename them, if that's the case.

Reply:
I assume you will push PR to finish line, let me know if that's not the case.